We All Lift Together: Mine and Underground Village

EdwardNashton opened the pull request:
Heavily reworked a Mine area. Added a 4 new buildings: Administrative, Cafeteria, Warehouse, Equipment Building.

![image](https://user-images.githubusercontent.com/19852466/167274667-562f2dd4-0116-48ae-b0de-cb64e60b2256.png)

![image](https://user-images.githubusercontent.com/19852466/167274678-ad0cf093-1387-46cf-8f81-e983afc04f4c.png)


Reworked ground level of Mines - now there's a few buildings in a caves, with 2nd floor.

![image](https://user-images.githubusercontent.com/19852466/167274688-eec0e68f-bf67-41a0-801f-3fda42642484.png)

![image](https://user-images.githubusercontent.com/19852466/167274699-100a96fa-3281-43ef-8c8f-cddda058db7e.png)


Slightly reworked an Underground village: replaced a couple of buildings, added more loot and a few more radscorpions. 
Slightly expanded a sewers - a new ruined tunnel that's kinda source of water for village.

![image](https://user-images.githubusercontent.com/19852466/167274713-9f7a6c9e-0c3e-47b8-a593-44f30f1465b1.png)

Commit:
We All Lift Together: Mine and Underground Village

Heavily reworked a Mine area. Added a 4 new buildings: Administrative, Cafeteria, Warehouse, Equipment Building.
Reworked ground level of Mines - now there's a few buildings in a caves, with 2nd floor.

Slightly reworked an Underground village: replaced a couple of buildings, added more loot and a few more radscorpions.

ProfessorPopoff commented:
Compilation error! You need to fix the formatting of the note you added. I posted how it should look in discord. 

The whole area looks really cool, now. A lot different than before and a welcome change in my eyes.

![image](https://user-images.githubusercontent.com/51188571/167317082-ece4d741-7df0-414e-83a5-18e19ac8e4c4.png)
We spoke a bit about loot in the discord. In general, keep T3/T4 stuff underground and behind stuff. This location could have T1/T2 tops. Along with this, I'd recommend limiting the loot type to just one or two. Giving a person the potential to get armour, a weapon, AND medical is scuffed.

![image](https://user-images.githubusercontent.com/51188571/167317118-409fde74-c71a-45ce-9898-a82388424edd.png)

![image](https://user-images.githubusercontent.com/51188571/167316953-305b5337-94e5-46dc-a7c9-15cb6aaf6cbc.png)
This little 'under construction' area probably is out of place, now that the mine is actually there. Especially with the fact that area looked operation, but has the road blocked off.

This is a really cool upper area, here. I love it . You need to put the mountain roofing helpers over all the open areas, though. Along with that, you should make that top bit look a bit more natural at that entrance... Even if it's just a slight dimple in one of the faces, it's REALLY square right now.
![image](https://user-images.githubusercontent.com/51188571/167317165-7defa9b9-19be-4574-a34c-dac19c231b1f.png)

![image](https://user-images.githubusercontent.com/51188571/167317250-4298a942-e188-4c97-a0b1-e6be991993af.png)
The center stairway isn't aligned with the stairs under it, and need to get moved over to the right by one tile.

![image](https://user-images.githubusercontent.com/51188571/167317197-3919b3f9-33fb-4a7d-8f08-1fc512053690.png)
I don't know if you think spore spawners will plant the mushrooms, but they don't. Just a heads up. It will be loose spores that haven't planted themselves for some reason, and I'd recommend they weren't just laying around.

![image](https://user-images.githubusercontent.com/51188571/167317217-fe8cb4e1-3d23-4643-9e83-db5f11c17f18.png)
Certain people may kill you if you use that TG tile. I advise against it, even if there's not an alternative right this second.
There's also this empty building thing up here that needs to get nicked. 
![image](https://user-images.githubusercontent.com/51188571/167317241-7101ad19-851c-47b5-878b-764bde57d414.png)

And for the big one. 
![image](https://user-images.githubusercontent.com/51188571/167317336-f61d2de7-1b0d-4a13-bfad-8be1058aaf0b.png)
Yeah. That's a city sized civilization down there.... We're going to have to snip a few buildings, and I'll likely send my recommended buildings in Discord. This makes me think that we need to have a 'Map vault', so that we can toss unused buildings into a reserve pit to use on other maps as a starter. 

Otherwise, the overall layout of it is really nice. Thanks once more.

ProfessorPopoff reviewed (dismissed):
Indentation error on the paper file. Should be structured like the others.

> /obj/item/paper/ms13/prewritten/crooked_note
    name = "paper - 'Crooked Note'"
    info = {"<i>I saw it in a blizzard. And it saw me. It haunts me for now, I know it. God, please, help...<B>
    </i>"}

Also don't forget the trailing newline. (An empty line after the code)

Commit:
We All Lift Together: fixing an issues

Cut some builidings in a village.
Made a caves on a north of village not so empty.
Removed a "under construction" building near the road, and signs on a road.
Fixed a shroom farm on z3-level.
Placed a roof helpers on a z3-level.
Changed a loot in a secret cave.
Changed a code of paper.
Fixed a stair issue on z2-level (ground level).

ProfessorPopoff committed:
Indentation on Paper.dm

Yeah we webediting. Keep scrolling.

ProfessorPopoff committed:
Adds pickaxes in, touches up loot and fixes some dirs

ProfessorPopoff commented:
Complaints addressed. I went ahead and added the new pickaxe in, even if it's got a placeholder inventory icon...

I touched up a few spots and fixed some things I missed. This is good to go.
